nupic/research/spatial_pooler.py

```diff
@@ -42,6 +43,57 @@ class InvalidSPParamValueError(ValueError):
 
 
 
+class _SparseMatrixCorticalColumnAdapter(object):
+  """ Many functions in SpatialPooler operate on a columnIndex but use an
+  underlying storage implementation based on a Sparse Matrix in which cortical
+  columns are represented as rows.  This can be confusing to someone trying to
+  follow the algorithm, confusing terminology between matrix math and cortical
+  columns.  This class is provided to abstract away some of the details of the
```

Comment:
Yeah it looks like `abc` only prevents it from being instantiated if some abstract method hasn't been overriden. I suppose fine as is.